File: examples/ruby/myro_test.rb
require 'Myro'
Myro::init
pic = Myro::takePicture
Myro::show pic
Myro::backward 1, 1
Myro::forward 1, 1
Myro::beep 1, 440
Myro::getPixels(pic).each do |pixel|
    r, g, b = Myro::getRGB pixel
    gray = (r + g + b)/3
    Myro::setRGB(pixel, gray, gray, gray)
end

File: examples/python/Misc/myro_test.py
from Myro import *
init()
pic = takePicture()

show(pic)
backward(1, 1)
forward(1, 1)
beep(1, 440)
for pixel in getPixels(pic):
    r, g, b = getRGB(pixel)
    gray = (r + g + b)/3
    setRGB(pixel, gray, gray, gray)
    
